runtime-config-linux.md

```diff
@@ -503,3 +503,17 @@ Its value is either slave, private, or shared.
 ```json
     "rootfsPropagation": "slave",
 ```
+
+## Disable new privileges
+
+Setting `disableNewPrivileges` to true prevents the processes in the container from gaining additional privileges.
```

Comment:
is there a reason to not call it noNewPrivileges so it matches the kernel version?